src/etc/inc/auth.inc

```diff
@@ -1371,6 +1371,11 @@ function ldap_backed($username, $passwd, $authcfg, &$attributes = array()) {
 		return false;
 	}
 
+	if (!isset($authcfg['ldap_allow_unauthenticated']) && $passwd == '') {
```

Comment:
Does this also affect someone submitting a password containing only whitespace? It seems like `empty($passwd)` might be a better choice here.

Reply:
unauthenticated LDAP binds only refers to binds with an empty string....So this line is fine. (I tested using a Windows AD to make sure).